### PR TITLE
feat(core/inlines): support attr-value type for elements

### PR DESCRIPTION
**Pull request by sidvishnoi**

### CODE REVIEW

File: src/core/inlines.js [hunk 1]
```diff
@@ -77,16 +77,28 @@ const inlineElement = /(?:\[\^[^^]+\^\])/; // Inline [^element^]
...
+  const [xrefType, xrefFor, textContent, linkingText] = (() => {
+    if (attrValue) {
+      const text = `${element} ${attribute} "${attrValue}"`;
+      return ["attr-value", `${element}/${attribute}`, text, attrValue];
+    } else if (attribute) {
+      return ["element-attr", element, attribute];
+    } else {
+      return ["element", null, element];
+    }
+  })();
```
Comment (sidvishnoi) [inline src/core/inlines.js:93]: If only JS supported switch-case as expressions...

File: tests/spec/core/inlines-spec.js [hunk 2]
```diff
@@ -334,6 +335,9 @@ describe("Core - Inlines", () => {
     const iframeAllowAnchor = doc.querySelector("#test2 a");
     expect(iframeAllowAnchor.textContent).toBe("allow");
     expect(iframeAllowAnchor.hash).toBe("#attr-iframe-allow");
+    const inputModeAnchor = doc.querySelector("#test3 a");
+    expect(inputModeAnchor.textContent).toBe(`html-global inputmode "text"`);
```
Comment (sidvishnoi) [inline tests/spec/core/inlines-spec.js:339]: For `[^ link/rel/manifest ^]` will render as `link rel "manifest"`.
Any better textContent?
Should we support renaming like  `[^ link/rel/manifest | link[rel="manifest"] ^]`

**Review by marcoscaceres — APPROVED** [review]
Looks good.

File: tests/spec/core/inlines-spec.js (hunk 2 above)
Comment (marcoscaceres) [reply tests/spec/core/inlines-spec.js]: as discussed, should just be manifest

**Pull request merged**